packages/node/src/store.ts

```diff
@@ -157,22 +157,15 @@ export class Store {
    * The app's installation is confirmed iff the store write operation
    * succeeds as the write operation's confirmation provides the desired
    * atomicity of moving an app instance from being proposed to installed.
-   * @param stateChannel
+   *
    * @param appInstance
    * @param appInstanceInfo
    */
-  public async updateChannelWithAppInstanceInstallation(
-    stateChannel: StateChannel,
+  public async saveRealizedProposedAppInstance(
     appInstanceIdentityHash: string,
     appInstanceInfo: AppInstanceInfo
   ) {
     await this.storeService.set([
-      {
```

Comment:
Can we change this behavior in another PR?

This needs to be looked at more closely in isolation.